eclipse/src/saros/Saros.java

```diff
@@ -200,6 +211,59 @@ public static boolean useHtmlGui() {
     return Platform.getBundle("saros.ui") != null && Boolean.getBoolean("saros.swtbrowser");
   }
 
+  private void shutdownPlugin(final boolean readAndDispatch) {
```

Comment:
Could you provide a javadoc and describe the effect of `readAndDispatch`?

Reply:
I guess I will remove it and do a check in the method instead if the display is available.